README.md

```diff
@@ -224,7 +224,7 @@ Observable<Entry<String, Point>> entries =
 
 Searching by distance on lat longs
 ------------------------------------
-See [LatLongExampleTest.java](src/test/java/com/github/davidmoten/rtree/LatLongExampleTest.java) for an example. The example depends on [*grumpy-core*](https://github.com/davidmoten/grumpy) artifact which is also on Maven Central.
+See [LatLongExampleTest.java](src/test/java/com/github/davidmoten/rtree/LatLongExampleTest.java) for an example. The example depends on [*grumpy-ogc*](https://github.com/davidmoten/grumpy) artifact which is also on Maven Central.
```

Comment:
This should remain `grumpy-core`. When you look at LatLongExampleTest.java there is a dependency on this class only :

```import com.github.davidmoten.grumpy.core.Position;```